spec_legacy/integration/application_spec.rb

```diff
@@ -26,7 +26,7 @@
 #
 # See doc/COPYRIGHT.rdoc for more details.
 #++
-require 'legacy_spec_helper'
+require_relative '../legacy_spec_helper'
```

Comment:
Why did you change this require? you can run legacy specs with `rspec -I legacy_spec <spec>` manually.